Makefile

```diff
@@ -206,10 +206,10 @@ destroy_%:
 
 .PHONY: destroy_tfstate
 destroy_tfstate:
-	find . -name terraform.tfstate.d -exec rm -rf {} \;
+	find . -name terraform.tfstate.d -exec rm -rf {} +
```

Comment:
With the previous approach, even though the command deleted the directories, it returned a non-zero exit code and make was aborted without executing the rest of the cleanup.